publisher/kafka_test.go

```diff
@@ -90,7 +89,7 @@ func TestKafka_ProduceBulk(suite *testing.T) {
 			client.On("Produce", mock.Anything, mock.Anything).Return(fmt.Errorf("Local: Unknown topic")).Once()
 			kp := NewKafkaFromClient(client, 10, "%s")
 
-			err := kp.ProduceBulk(collection.CollectRequest{SendEventRequest: &pb.SendEventRequest{Events: []*pb.Event{{EventBytes: []byte{}, Type: topic}}}}, make(chan kafka.Event, 2))
+			err := kp.ProduceBulk([]*pb.Event{{EventBytes: []byte{}, Type: topic}}, "group-1", make(chan kafka.Event, 2))
```

Comment:
`group-1` should be declared as constant or global variable as it has been used at many places

Reply:
changed